devtools/client/README.md

```diff
@@ -1,3 +1,66 @@
-# devtools-client
+# @player-tools/devtools-client
 
-Package responsible for providing the common constructs (TODO: maybe even including redux) responsible for managing state and consuming events and RPCs.
+[![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](./LICENSE)
+
+The `@player-tools/devtools-client` exposes the Panel containing the ReactPlayer responsible for running the content sent by the Player devtool plugins on the inspected Player UI instance and the navigation bar.
+
+You can check how to use it in the [browser-extension](TODO) and [Flipper desktop App client](TODO).
+
+## Overview
```

Comment:
Might be out of scope for this PR but we should also put this info on our doc site.

Reply:
Issue to add the new dev tools to the Player docs: https://github.com/player-ui/player/issues/332

Also, I added the link to the [browser-extension](https://github.com/player-ui/browser-devtools), and the Flipper desktop app assuming the merge.